src/utils/events/formatEventFromServer.js

```diff
@@ -0,0 +1,69 @@
+import { formatEventTypeFromServer, formatOwnerFromServer } from './index';
+
+/**
+ * Normalizes the event object returned from the API server
+ *
+ * @param {Object} input
+ * @param {booelan} input.allow_others_to_trigger Whether or not non-owners can trigger the event
+ * @param {string} input.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.deleted_at ISO 8601 Extended Format date/time string
+ * @param {Object} input.EventType
+ * @param {string} input.EventType.client_id The ID of the client to which the event type belongs
+ * @param {string} input.EventType.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.EventType.description A description of the event type
+ * @param {string} input.EventType.id UUID corresponding with the event type
+ * @param {string} input.EventType.is_realtime_enabled
+ * @param {number} input.EventType.level The level of the event type
+ * @param {string} input.EventType.name The name of the event type
+ * @param {string} input.EventType.slug The slug of the event type
+ * @param {string} input.EventType.updated_at ISO 8601 Extended Format date/time string
+ * @param {string} input.event_type_id UUID corresponding with the event type
+ * @param {number} input.facility_id The Facility to which the event belongs
+ * @param {string} input.id UUID corresponding with the event
+ * @param {boolean} input.is_public Whether or not the event is public
+ * @param {string} input.name The name of the event
+ * @param {string} input.organization_id UUID corresponding with the organization
+ * @param {string} input.Owner
+ * @param {string} input.Owner.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.Owner.email The email address of the owner
+ * @param {string} input.Owner.first_name The first name of the owner
+ * @param {string} input.Owner.id The ID of the user who owns the event
+ * @param {string} input.Owner.is_machine_user
+ * @param {string} input.Owner.last_name The last name of the owner
+ * @param {string} input.Owner.updated_at ISO 8601 Extended Format date/time string
+ * @param {string} input.owner_id The ID of the user who owns the event
+ * @param {string} input.topic_arn The Amazon Resource Name (ARN) associated with the event
```

Comment:
Are any of these `input`s optional in the sense that it's possible the value might not be there or will all of these be present 100% of the time? If a value may be optional then we format it by adding `[]`s around it. See [here](https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/utils/assets/formatAssetAttributeFromServer.js#L13)

Reply:
Yeah, I have tests for ensuring that nothing crashes when those optional fields aren't present. I honestly just forgot to do the brackets around them when doing the documentation. I'll fix that up! Thx!